### PR TITLE
Added qps config files

### PR DESCRIPTION
**Pull request by jshaofuturewei**

### CODE REVIEW

File: globalscheduler/cmd/scheduler_process/scheduler_process.go [hunk 1]
```diff
@@ -88,7 +89,7 @@ func StartSchedulerController() {
 	clusterInformerFactory := clusterinformers.NewSharedInformerFactory(clusterClientset, time.Second*30)
 	clusterInformer := clusterInformerFactory.Globalscheduler().V1().Clusters()
 
-	schedulerController := scheduler.NewSchedulerController(kubeClientset, apiextensionsClient, schedulerClientset, clusterClientset, schedulerInformer, clusterInformer)
+	schedulerController := scheduler.NewSchedulerController(kubeClientset, apiextensionsClient, schedulerClientset, clusterClientset, schedulerInformer, clusterInformer, *configFile)
```
Comment (cathyhongzhang) [inline globalscheduler/cmd/scheduler_process/scheduler_process.go:92]: The scheduler controller and the scheduler crd are created in gs-controller-manager.go. There is no need to repeat these in this file.

Comment (jshaofuturewei) [reply globalscheduler/cmd/scheduler_process/scheduler_process.go:92]: We are using hq schedulers now. The process file is no longer needed.

**Review by cathyhongzhang — APPROVED** [review]
LGTM now

**Review by kimeunju108 — APPROVED** [review]
LGTM

**Pull request merged**